### ISSUE TITLE
Multiple pages document

### ISSUE THREAD

**Issue by victor-ab** [naiveHobo/InvoiceNet#41]
Hi, how would you treat/use invoicenet with multiple pages documents?

**Comment by dajor** [comment]
How is the idea the addition pages will be in the png. Idea is to make the png just larger

**Comment by kraakman** [comment]
I don't think that will work because you then lose the spatial locations of a single page, for example the bottom of page 1 is no longer the bottom in the concatenated img, but now somewhere in the middle. Plus the model requires specific image input of 128 * 128 pixels, so resizing the concatenated image will also hurt performance..

**Comment by victor-ab** [comment]
> I don't think that will work because you then lose the spatial locations of a single page, for example the bottom of page 1 is no longer the bottom in the concatenated img, but now somewhere in the middle. Plus the model requires specific image input of 128 * 128 pixels, so resizing the concatenated image will also hurt performance..

Agreed. Besides that, the layout will change a lot and the text density on the page will be much higher.
